src/js/main.js

```diff
@@ -52,10 +52,15 @@ Site.Menu = {
 
   onItemClick: function(item, event) {
     var _this = this;
+    var href = $(item).children('a').attr('href');
+    var hash = href.substr(1);
 
     $('.header-menu-active').removeClass('header-menu-active');
     $(item).addClass('header-menu-active');
-  }
+
+    $('.page-content').removeClass('page-content-active');
+    $('.page-content[data-slug="' + hash + '"]').addClass('page-content-active');
```

Comment:
mmm can you point out where they asked for full scroll? maybe on for mobile? tho i anticipated for mobile we would just show all the `.page-content` divs.

in the designs its not full scroll. check out the .ai:

![screen shot 2017-05-09 at 3 19 37 pm](https://cloud.githubusercontent.com/assets/7727722/25870810/19b3aac2-34cb-11e7-92ce-24e55628761f.png)